mercury_api/rpc/views.py

```diff
@@ -36,11 +37,22 @@ def get(self, job_id=None):
         projection = self.get_projection_from_qsa()
         if job_id is None:
             data = self.rpc_client.get_jobs(projection or {'instruction': 0})
+            for job in data['jobs']:
+                job_id = job['job_id']
+                add_links(job, [
+                    f'/{job_id}',
+                    f'/status/{job_id}',
+                    f'/tasks/{job_id}'
+                ])
```

Comment:
curious what the output to the client looks like; hopefully it's something like:
```json
{
   "links": [
       {
           "name": "job",
           "link": "https://..../<job-id>/..."
       },
   ]
}
```

Reply:
it's just a list of urls